lib/iris/tests/unit/experimental/representation/test_CubeListRepresentation.py

```diff
@@ -7,15 +7,14 @@
 
 from html import escape
 
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests
-
-from iris.cube import CubeList
 import iris.tests.stock as stock
-
+from iris.cube import CubeList
 from iris.experimental.representation import CubeListRepresentation
 
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests  # isort:skip
```

Comment:
Needs moving? Not being imported first.

Reply:
The `html` modules doesn't import `mpl`... so that safe, but I'll just move it anyways since I'm here :+1: